### PR TITLE
Climber and LevelClimber commands

### PR DESCRIPTION
**Pull request by 2Train395**

### CODE REVIEW

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -23,72 +25,76 @@
...
   /**
-   * This function is called periodically during operator control.
-   */
+  * This function is called periodically during operator control.
+  */
   @Override
   public void teleopPeriodic() {
     Scheduler.getInstance().run();
+    SmartDashboard.putNumber("RobotRoll", gyro.getRoll());
+    //elevator.test(oi.getElevatorThrottle());
```
Comment (dhoizner) [inline src/main/java/frc/robot/Robot.java]: remove commented code?

File: src/main/java/frc/robot/RobotMap.java [hunk 2]
```diff
@@ -16,5 +16,9 @@
 
     //Intake
     public static final int rollerIntakeTalon = 11;
+
+    //Climber
+    public static final int leadScrewTalon = 7;
```
Comment (dhoizner) [inline src/main/java/frc/robot/RobotMap.java]: looking at these, we might want to namespace our id variables, i.e.: `intakeRollerTalon`, `climberLeadScrewTalon`, `climberWheelPodTalon`.

also, adding `Id` to the end of these will make name collisions when instantiating them less likely and probably make what they are clearer.

File: src/main/java/frc/robot/commands/LevelRobot.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj.PIDController;
+import edu.wpi.first.wpilibj.command.Command;
+import frc.robot.Robot;
+
+public class LevelRobot extends Command {
+
+  //P is negative because nose down should lower elevator and nose down is negative angle
+  PIDController pidController = new PIDController(1.0/20.0, 0, 1.0/200.0, Robot.gyro.getRollSource(), Robot.elevator.levelElevator());
```
Comment (dhoizner) [inline src/main/java/frc/robot/commands/LevelRobot.java]: what are these magic numbers?

File: src/main/java/frc/robot/subsystems/Climber.java [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+ */
+public class Climber extends Subsystem {
+
+  WPI_TalonSRX leadScrew = Robot.speedControllerMap.getTalonByID(RobotMap.leadScrewTalon);
+  WPI_TalonSRX wheelPod = Robot.speedControllerMap.getTalonByID(RobotMap.wheelPodTalon);
+
+  double fixedSpeedForTesting = 0;
+
+  public Climber() {
+    Preferences.getInstance().putDouble("climberMaxSpeed", 0.0);
```
Comment (dhoizner) [inline src/main/java/frc/robot/subsystems/Climber.java]: having this in here for now (tuning) is okay, but lets add a TODO for cleaning this up before competition.

File: src/main/java/frc/robot/subsystems/Elevator.java [hunk 5]
```diff
@@ -53,27 +59,27 @@
...
+    leaderConfig.motionAcceleration = 3500; //TODO
     
     leaderConfig.continuousCurrentLimit = 30;
     leaderConfig.peakCurrentLimit = 30;
-    leaderConfig.peakCurrentDuration = 500;  
+    leaderConfig.peakCurrentDuration = 0;  
     
-    leaderConfig.closedloopRamp = 0.15;
-    leaderConfig.openloopRamp = 0.15;
+    leaderConfig.closedloopRamp = 0.5;
```
Comment (dhoizner) [inline src/main/java/frc/robot/subsystems/Elevator.java]: these should probably be revisited with the tensioned chain?

File: src/main/java/frc/robot/RobotMap.java (hunk 2 above)
Comment (HPrickettMorgan) [reply src/main/java/frc/robot/RobotMap.java]: Makes sense, I'll put up another PR after this to do that.

File: src/main/java/frc/robot/subsystems/Climber.java (hunk 4 above)
Comment (HPrickettMorgan) [reply src/main/java/frc/robot/subsystems/Climber.java]: It's actually fine to be 100% on both, I'll just take it out.

File: src/main/java/frc/robot/subsystems/Elevator.java (hunk 5 above)
Comment (HPrickettMorgan) [reply src/main/java/frc/robot/subsystems/Elevator.java]: Sure

Comment (HPrickettMorgan) [reply src/main/java/frc/robot/subsystems/Elevator.java]: I'll add a TODO

**Pull request merged**